server/leafnode.go

```diff
@@ -80,9 +90,12 @@ func validateLeafNode(o *Options) error {
 	return nil
 }
 
-func (s *Server) reConnectToRemoteLeafNode(remote *RemoteLeafOpts) {
-	// TODO(dlc) - We will use route delay semantics for now.
-	delay := DEFAULT_ROUTE_RECONNECT
+func (s *Server) reConnectToRemoteLeafNode(remote *leafNodeCfg) {
+	delay := s.getOpts().LeafNode.ReconnectInterval
+	if delay == 0 {
+		delay = DEFAULT_LEAF_NODE_RECONNECT
+	}
+	delay += time.Duration(rand.Intn(100)) * time.Millisecond
```

Comment:
There will not be tons of these, jitter may not be necessary and would probably not all hit a single server anyway.

Reply:
Yep, will remove